### DISCUSSION TITLE
已经6.18号了，请问中文版课程今天会更新吗？

### DISCUSSION THREAD

**Discussion by coldlarry** [mlc-ai/mlc-zh#8] [Q&A] [answered]
非常期待大佬的讲课，很焦急呀~

**Comment by MasterJH5574** [comment]
第一讲的中文视频已经发布至[B站](https://www.bilibili.com/video/bv15v4y1g7EU)，欢迎前来！我们会很快添加到课程主页上。不过目前还没有中文字幕

**Reply by MasterJH5574** [reply]
我们已经将第一讲的视频、课件与英文笔记更新至课程主页 https://mlc.ai/summer22-zh/schedule ，请大家保持关注！

**Comment by coldlarry** [comment, marked as answer]
感谢老师和助教团队，已经找到了中文视频，而且发现是用国语讲课的。很佩服这样的授课态度，会和身边的小伙伴继续支持哦！

**Reply by lambda7xx** [reply]
> 非常期待大佬的讲课，很焦急呀~

中美之间是有时差的，
